addons/gunbag/initSettings.sqf

```diff
@@ -0,0 +1,10 @@
+// CBA Settings [ADDON: ace_gunbag]:
+
+[
+    QGVAR(swapGunbagEnabled), "CHECKBOX",
+    [LSTRING(SwapGunbagEnabled_DisplayName), LSTRING(SwapGunbagEnabled_Description)],
+    LSTRING(DisplayName_Settings),
+    false, // default value
+    true, // isGlobal
+    {[QGVAR(swapGunbagEnabled), _this] call EFUNC(common,cbaSettings_settingChanged)}
```

Comment:
Isn't `cbaSettings_settingChanged` required only for settings that were converted from old system? If the setting is new it's not needed. Can someone from @ace3mod team confirm?
```suggestion
    true // isGlobal
```